fx_crash_sig/crash_processor.py

```diff
@@ -61,9 +61,9 @@ def symbolicate(self, payload):
         else:
             symbolicated = self.symbolicator.symbolicate(crash_data)
 
-        return self.__postprocess(payload, symbolicated)
+        return self._postprocess_symbolicated(payload, symbolicated)
 
-    def __postprocess_symbolicated(self, payload, symbolicated):
+    def _postprocess_symbolicated(self, payload, symbolicated):
```

Comment:
We shouldn't be using `__` for methods. It makes testing and a bunch of other things harder and "private methods" don't have the same meaning in Python that they do for other languages.